maxim/tests/test_crewai.py

```
@@ -0,0 +1,98 @@
+from dotenv import load_dotenv
+from crewai import Agent, Task, Crew, Process
+
+from maxim import Maxim
+from maxim.logger.crewai import instrument_crewai
+
+load_dotenv()
+
+class TestCrewAI(unittest.TestCase):
+    """Test class for CrewAI integration with Maxim logging."""
+
+    def setUp(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)